src/tasks/download-paportal/download-paportal-v2/index.ts

```diff
@@ -34,6 +34,7 @@ export async function main(): Promise<void> {
       websiteId: parameterMap["WebsiteId"],
       overwrite: parameterMap["Overwrite"],
       excludeEntities: parameterMap["ExcludeEntities"],
+      modelVersion: parameterMap["ModelVersion"],
```

Comment:
You'll also need to add the parameter to the `tasks.json` in the same directory.

Reply:
Added